ManiVault/src/private/LearningPageContentWidget.cpp

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later 
+// A corresponding LICENSE file is located in the root directory of this source tree 
+// Copyright (C) 2023 BioVault (Biomedical Visual Analytics Unit LUMC - TU Delft) 
+
+#include "LearningPageContentWidget.h"
+
+#include <QDebug>
+
+using namespace mv::gui;
+
+LearningPageContentWidget::LearningPageContentWidget(QWidget* parent /*= nullptr*/) :
+    PageContentWidget(Qt::Vertical, parent),
+    _videosWidget(this),
+    _tutorialsWidget(this),
+    _examplesWidget(this),
+    _pluginResourcesWidget(this),
+    _showVideosAction(this, "Show videos", true),
+    _showTutorialsAction(this, "Show tutorials (coming soon)", false),
+    _showExamplesAction(this, "Show examples (coming soon)", false),
+    _showPluginResourcesAction(this, "Show plugin resources"),
```

Comment:
I think we should toggle this on by default as well, like `_showVideosAction`.

Reply:
The the tutorials and examples section have no content atm.

Reply:
I tried to just highlight `_showPluginResourcesAction`:

```
_showPluginResourcesAction(this, "Show plugin resources", true),
```

Reply:
(The plugin resources didn't show - like in the video - for me and I had to toggle them on.)

Reply:
Ah like so! I get it, yeah sure, I will enable it :)